tests/Tags/AssetsTagTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace Tests\Tags;
+
+use Illuminate\Support\Facades\Storage;
+use Statamic\Assets\Asset;
+use Statamic\Assets\AssetCollection;
+use Statamic\Assets\AssetContainer;
+use Statamic\Facades;
+use Statamic\Facades\Parse;
+use Tests\TestCase;
+
+class AssetsTagTest extends TestCase
+{
+    public function setUp(): void
+    {
+        parent::setUp();
+
+        config(['filesystems.disks.test' => [
+            'driver' => 'local',
+            'root' => __DIR__.'/tmp',
+        ]]);
+
+        Storage::fake('test');
+        Storage::fake('dimensions-cache');
+    }
+
+    private function tag($tag, $data = [])
+    {
+        return (string) Parse::template($tag, $data);
+    }
+
+    /** @test */
+    public function it_outputs_assets()
+    {
+        $container = (new AssetContainer)
+            ->handle('test_container')
+            ->disk('test');
```

Comment:
This isn't a mock, but this is what you're providing to AssetContainer::find() below.